bikeshed/boilerplate.py

```diff
@@ -775,18 +775,96 @@ def addIDLSection(doc: t.SpecT) -> None:
         del el.attrib["id"]
     h.addClass(doc, container, "highlight")
 
+bookmarkCss = """
+.material-symbols-outlined {
+  font-variation-settings: 'FILL' 0, 'wght' 400, 'GRAD' 0, 'opsz' 24
```

Comment:
Mix of indentations here and in the script. Use a consistent 4-space indent to match the rest of the file, and start the script from zero indent.

Reply:
Fixed, I think.  The bookmarkCss and bookmarkScript variable are defined at the top level, with content that is now indented.